bug-trends.py

```diff
@@ -184,22 +214,38 @@ def main():
         print(e)
         sys.exit(1)
 
-    #bzapi.bug_autorefresh = True
+    # bzapi.bug_autorefresh = True
+
+    if args.bugzilla:
+        try:
+            creation_time = bzapi.getbug(args.bugzilla).creation_time
```

Comment:
we query api here and then again on L190. I'm not very familiar with the bugzilla module but isn't there a way to get all data with one query? Bugzilla API doesn't belong among fastest APIs I've seen :)